### ISSUE TITLE
Documentation for .Net Core

### ISSUE THREAD

**Issue by davidsk** [Breeze/breeze.server.net#72]
There doesn't seem to be any documentation on using Breeze with .Net Core outwith the github issues.  I have managed to coax it into life by inspecting efforts by others.  I appreciate that it's yet to make v1.0 but there are some significant changes and little guidance as to how to use in a .Net Core app.

**Comment by sp00ky** [comment]
Very much agree that this is an issue.  For anybody finding this before such docs are produced, take a look at this github project:

https://github.com/mikemichaelis/dnc

**Comment by plwade** [comment]
I've got this working https://github.com/mikemichaelis/dnc , but I'm struggling to get a Breeze/Angular client working with it.
Can anybody point me in the right direction?

**Comment by ganySA** [comment]
DNC seems outdated, i cannot seem to get the routes working and doesnt really have any steps - any chance we could have a few page wiki to setup that works on 2.2?

**Comment by steveschmitt** [comment]
Yes, we've been reallllly slow with this, but we'll get a sample app and some docs going in the near future.

**Comment by ganySA** [comment]
Thanks for the speedy reply... 
I know you guys do this as a huge value add to the community and with the early versions of breeze the website was great in terms of documentation - I think with .core it’s just become almost impossible to guess / workout how to put it all together - even nuget is clogged with packages no one knows what to install to make it work + multiple versions.

I’m sure you guys have a huge todo list, but this one item is killing the adoption of breeze and filling up the issue list.

Without sound pessimistic we have been hearing about .core early since 2017... 

I think a little github doc on 5-10 steps to get it working, not too much of an ask for such an amazing product. 

my 2c

**Comment by plwade** [comment]
I've managed to get breeze working with .net core 2. It was a real trial and error exercise..... I'm on holiday at the moment, but will give you a few pointers when I'm back (probably in three weeks time).

Kind Regards,
Paul

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by ganySA** [comment]
@plwade Did you manage to get .expand to work on breeze queries?

Could you share your configuration.

**Comment by plwade** [comment]
@gany just got back from holiday, so will try and send you something today.

Paul

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by ganySA** [comment]
plwade: i have managed to get breeze working - i just cannot get my queries to execute .expand('....') to work. I either get circular reference errors in the json, then i disabled this. Now i get other entity errors. I am not sure it works.

**Comment by steveschmitt** [comment]
I finally built that demo project, and wrote the steps to create it from scratch.
https://github.com/Breeze/northwind-core-ng-demo
The demo is a .NET Core 2.2 web API, with Angular 8 on the front end.
I hope it helps.

**Comment by steveschmitt** [comment]
Demo project URL is now https://github.com/Breeze/northwind-demo 
It includes a .NET 5 demo.
